### PR TITLE
Add env var ADDITIONAL_PYTHON_PATH to load custom python packages

### PR DESCRIPTION
**Pull request by aucahuasi**
Sometimes we want to load custom python packages into the GAK container.

With this PR we just need to mount the volume to the local directory that contains the custom packages and export it as the `ADDITIONAL_PYTHON_PATH` env var.

For example, using docker compose:

```yaml
services:
  graph-app-kit-service:
    image: graphistry/graph-app-kit-st:latest-11.5
    environment:
        ADDITIONAL_PYTHON_PATH: "/mnt"
    volumes:
      - /home/user/some_python_pkgs:/mnt
```

### CODE REVIEW

File: src/docker/entrypoint.sh [hunk 1]
```diff
@@ -20,7 +20,7 @@ echo "Log level (LOG_LEVEL): $LOG_LEVEL"
 echo "Graphistry user (GRAPHISTRY_USERNAME): $GRAPHISTRY_USERNAME"
 echo ""
 
-
+export PYTHONPATH=$PYTHONPATH:$ADDITIONAL_PYTHON_PATH
```
Comment (lmeyerov) [inline src/docker/entrypoint.sh:23]: protect with `if [ -z ...` or equiv ?

**Review by lmeyerov — CHANGES_REQUESTED** [review]
see comment

also, i think we should document an example of using, like adding a different version of pygraphistry + a new dependency like faker

File: src/docker/entrypoint.sh (hunk 1 above)
Comment (aucahuasi) [reply src/docker/entrypoint.sh]: Good catch, thanks! Done!

File: docs/additional-packages.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        [
+          "CMD",
+          "curl",
+          "-Lf",
+          "http://localhost:8501/${BASE_PATH}healthz"
+        ]
+      interval: 30s
+      timeout: 30s
+      retries: 10
+      start_period: 10s
```
Comment (lmeyerov) [inline docs/additional-packages.md:50]: i think we can remove the full version and just show the below

File: src/docker/entrypoint.sh [hunk 3]
```diff
@@ -20,7 +20,10 @@ echo "Log level (LOG_LEVEL): $LOG_LEVEL"
 echo "Graphistry user (GRAPHISTRY_USERNAME): $GRAPHISTRY_USERNAME"
 echo ""
 
-
+if [ -n "$ADDITIONAL_PYTHON_PATH" ]; then
+    export PYTHONPATH=$PYTHONPATH:$ADDITIONAL_PYTHON_PATH
+    echo "Using ADDITIONAL_PYTHON_PATH=$ADDITIONAL_PYTHON_PATH"
```
Comment (lmeyerov) [inline src/docker/entrypoint.sh:25]: I think our imports should go first, not last, right?

File: docs/additional-packages.md (hunk 2 above)
Comment (aucahuasi) [reply docs/additional-packages.md]: Thanks, done!

File: src/docker/entrypoint.sh (hunk 3 above)
Comment (aucahuasi) [reply src/docker/entrypoint.sh:25]: I see, done!

**Review by lmeyerov — APPROVED** [review]
tx!

**Pull request merged**